docs/DeveloperGuide.md

```diff
@@ -1,18 +1,133 @@
-# Developer Guide
+# Developer Guide - Diet Manager
 
-## Design & Implementation
+## Content
+1. [Design](#design)
+2. [Implementation](#implementation)
+2. [Product Scope](#product-scope)
+3. [User Stories](#user-stories)
+4. [Non-Functional Requirements](#non-functional-requirements)
+5. [Glossary](#glossary)
+6. [Instructions for Manual Testing](#instructions-for-manual-testing)
+
+## Design
 
 {Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
 
+### Architecture
+
+![ArchitectureDiagram](/docs/images/Architecture.png)
```

Comment:
The image is not rendered on the github pages

Reply:
Same! I face similar issue! 😢  
I tried accessing the DG directly through github/docs/DeveloprGuide.md